src/toggle/toggle.ts

```diff
@@ -27,26 +27,21 @@ export type ToggleState = {
   selector: 'ais-toggle',
   template: `
     <div [class]="cx()">
-      <ul [class]="cx('list')">
-        <li
-          [class]="cx('item')"
-          (click)="handleClick($event)">
-          <label [class]="cx('label')">
-            <input
-              [class]="cx('checkbox')"
-              type="checkbox"
-              value="{{state.value.name}}"
-              [checked]="state.value.isRefined"
-            />
+      <label [class]="cx('label')">
+        <input
+          [class]="cx('checkbox')"
+          type="checkbox"
+          value="{{state.value.name}}"
+          [checked]="state.value.isRefined"
+          (change)="handleChange($event)"
+        />
 
-            <span [class]="cx('labelText')">
-              {{label || state.value.name}}
-            </span>
+        <span [class]="cx('labelText')">
+          {{label || state.value.name}}
+        </span>
 
-            <span [class]="cx('count')">{{state.value.count}}</span>
-          </label>
-        </li>
-      </ul>
+        <span [class]="cx('count')">{{state.value.count}}</span>
```

Comment:
Side note: this `count` element is currently not in the spec because it wasn't updated after some users requested to add it. It's implemented in Vue InstantSearch and Angular InstantSearch. We need to make sure to add it before their respective major releases.